x-pack/legacy/plugins/ml/public/application/timeseriesexplorer/timeseriesexplorer.js

```diff
@@ -319,7 +320,11 @@ export class TimeSeriesExplorer extends React.Component {
       to: selection.to.toISOString(),
     };
 
-    if (isEqual(this.props.zoom, zoomState) && this.state.focusChartData !== undefined) {
+    if (
+      isEqual(this.props.zoom, zoomState) &&
+      this.state.focusChartData !== undefined &&
+      this.props.previousRefresh === this.props.lastRefresh
```

Comment:
I find this naming a bit confusing, previous and last refreshes. Maybe stick to the standard previous / current?

Reply:
Happy to discuss for a follow up but not sure it's worth changing it in this bugfix PR. We use `lastRefresh` already in various places across the code base so for consistency it needed to be changed everywhere. Also not sure about naming it `current` because it's a timestamp and will always be a value from the past when it happened :).  I used `previousRefresh` to avoid the even worse `previousLastRefresh` (see the diff further down) :). What do you think ok to leave as is?

Reply:
An alternative came to mind: `lastRefresh` (keep as is for consistency) and `secondLastRefresh`?